core/src/main/java/org/elasticsearch/index/query/PrefixQueryParser.java

```diff
@@ -30,7 +30,9 @@
  */
 public class PrefixQueryParser implements QueryParser<PrefixQueryBuilder> {
 
-    private static final ParseField NAME_FIELD = new ParseField("_name").withAllDeprecated("query name is not supported in short version of prefix query");
+    public static final ParseField NAME_FIELD = new ParseField("_name").withAllDeprecated("query name is not supported in short version of prefix query");
+    public static final ParseField PREFIX_FIELD = new ParseField("value", "prefix");
```

Comment:
Putting myself into the user's shoes: I personally would prefer having one and only one non-deprecated way of specifying something - makes it easier to compare different implementations without worrying what the subtle difference is when using either option. Not sure which version to keep here though.